src/components/ConfirmDialog.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+
+const ConfirmDialog = (props) => {
+  const {
+    cancelButton,
+    children,
+    confirmButton,
+    onCancel,
+    onConfirm,
+    open,
+    title,
+  } = props;
+
+  return (
+    <Dialog
+      open={open}
+      onClose={onCancel}
+      aria-labelledby="confirm-dialog"
+    >
+      <DialogTitle id="confirm-dialog">{title}</DialogTitle>
+      <DialogContent>{children}</DialogContent>
+      <DialogActions>
+        <Button variant="contained" onClick={onCancel} color="secondary">
+          {cancelButton}
+        </Button>
+        <Button variant="contained" onClick={onConfirm} color="primary">
+          {confirmButton}
+        </Button>
```

Comment:
Quick nitpick from the UX review, if we wanted to go ahead and make this change while we're here:

> Change buttons to say "Yes, remove program." and "No, don't remove program." Show them in that order, with the "no" button the primary color and the "yes" button the grey.  (Less destructive option always goes on right)